.github/workflows/build-test-distribute.yaml

```diff
@@ -100,18 +81,15 @@ jobs:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
         with:
           fetch-depth: 0
-      - name: "Check if should run on all arch/os combinations"
+      - name: "Maybe set full matrix"
         if: github.event_name == 'push' || contains(github.event.pull_request.labels.*.name, 'ci/run-full-matrix')
-        run: |
-          echo 'RUN_FULL_MATRIX=true' > .run-full-matrix
-          echo 'RUN_FULL_MATRIX=true' >> $GITHUB_ENV
-      - name: "Set switches based on if run full matrix"
-        if: ${{ env.RUN_FULL_MATRIX == 'true' }}
         id: set-full-matrix-switches
         run: |
           echo 'ENABLED_GOARCHES=arm64 amd64' >> $GITHUB_ENV
           echo 'ENABLED_GOOSES=linux darwin' >> $GITHUB_ENV
-      - name: "Check if force push"
+      - run: |
```

Comment:
maybe worth always setting a `name` if what the `run` is doing isn't super obvious